dockerfiles/buildall.sh

```diff
@@ -38,6 +39,7 @@ if [[ -n ${INPUT_BUILDTYPE} ]] ; then
 else
   BUILD_LABEL=${BUILD_TAG}
   BUILD_TYPE=RelWithDebInfo
+  USE_CPPUNIT=0
```

Comment:
just to check - you're intentionally turning off, in the default case, when no INPUT_BUILDTYPE has been specified?
i.e. it's left on if the INPUT_BUILDTYPE is anything, including release?

Reply:
My goal was that the "standard" builds with no suffix on the build label did not include cppunit (though to be honest it's arguable whether they should and probably harmless for them to do so). Any "non-standard" build is for developer use and should include. I don't have strong feelings though - we could just set on all the time and simplify things significantly

Reply:
@richardkchapman - minor image size increase with cppunit libs included? probably insignificant.

I think it's fine as it stands.